lib/tizen_build_target.dart

```diff
@@ -273,9 +273,10 @@ class TizenPlugins extends Target {
         sharedLib.copySync(outputDir.childFile(sharedLib.basename).path);
 
         // copy binaries that plugin depends on
-        final Directory pluginLibDir = pluginDir
-            .childDirectory('lib')
-            .childDirectory(arch == 'arm' ? 'armel' : 'i586');
+        final String pluginArch =
+            arch == 'arm' ? 'armel' : (arch == 'x86' ? 'i586' : arch);
+        final Directory pluginLibDir =
+            pluginDir.childDirectory('lib').childDirectory(pluginArch);
```

Comment:
@HakkyuKim For Tizen plugins with shared libs, which directory should I place arm64 libs?

- arm: armel
- x86: i586
- arm64: aarch64?

Reply:
aarch64. Tizen CLI uses that name internally.

Tizen CLI uses the "architecture" attribute in the rootstrap xml file to replace {BUILD_ARCH} variable. This may allow us to use the names arm and x86, but the attribute value is used in several places internally so we should leave the names to armel and i586 at the moment.